compiler/passes/expandExternArrayCalls.cpp

```diff
@@ -97,6 +100,14 @@ void expandExternArrayCalls() {
 
     if (fcopy) {
       SET_LINENO(fn);
+      Expr* parentScope = fn->defPoint->parentExpr;
+      if (cptrScopes.count(parentScope) == 0) {
+        BlockStmt* useBlock = buildChapelStmt(new UseStmt(new UnresolvedSymExpr("CPtr"), "",
+                                                        true));
+        fn->defPoint->insertAfter(useBlock);
```

Comment:
Not that it matters for correctness, but `insertBefore` sounds more right to me

Reply:
I am assuming that this is happening before scopeResolve (it must be, right?), so it might help AST dumps look cleaner to make it `insertBefore`

Reply:
I used insertAfter because I sometimes worry that if I don't know the context well enough to be certain that something isn't the first thing in its list, that there's a chance what I insert will be orphaned.  But that is based on superstition and fear than certainty or fact.  And I knew that it wouldn't/shouldn't matter whether the `use` came before or after the statement in question, semantically.  If you know better than I do that it's safe to user `insertBefore`, I can switch it.

Reply:
I see. I sure hope that what you fear cannot happen, but I am not certain that it won't. And I have no problem keeping this as-is.

Reply:
If `insertBefore` is written well, I agree with you that it shouldn't be problematic.  I was coding this up late at night, so being cautious.  :)